cloud/azure/bin/lib/storage.sh

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+
+readonly BLOB_DATA_CONTRIBUTOR_GUID="ba92f5b4-2d11-453d-a403-e96b0029c9fe"
+
+#######################################
+# Assign the role 'Storage Blob Data Contributor' to the current user
+#######################################
+function storage::assign_storage_blob_data_contributor_role_to_user() {
+  local USER_ID=$(az ad signed-in-user show --query objectId -o tsv)
+  local SUBSCRIPTION_ID=$(az account show --query id -o tsv)
+  local ROLE_ASSIGNMENTS=$(az role assignment list --assignee ${USER_ID})
+  if [[ ("Storage Blob Data Contributor" == *$ROLE_ASSIGNMENTS*) ]];
+  then 
+    echo "Current user already has Storage Blob Data Contributor role"
+  else
+    az role assignment create \
+      --role  "${BLOB_DATA_CONTRIBUTOR_GUID}" \
+      --scope "subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${resource_group}" \
```

Comment:
question (blocking): how is `resource_group` set?

Reply:
I guess this was working assuming that the calling script had set the value. changed it to explicitly pass in the resource group as an argument.